news letter draft (#1924)
rcgsheffield/sheffield_hpc: newsletter/20231201.rst

.. _nl20231201:

HPC Newsletter 2023-12-01
=========================

ShARC Decommissioned
---------------------

> - As of November 30th, we have officially decommissioned our oldest High-Performance Computing (HPC) cluster, ShARC. For reference, we have archived `it here <https://docs.hpc.shef.ac.uk/en/latest/decommissioned/index.html/>`_.

Reviewer: ```suggestion
- As of November 30th, we have officially decommissioned our oldest High-Performance Computing (HPC) cluster, ShARC. For reference, we have archived `ShARC's documentation here <https://docs.hpc.shef.ac.uk/en/latest/decommissioned/index.html/>`_.
```

Reviewer: Also, reference decommissioned cluster page via Sphinx link, not absolute URL?

- Support remains available for users needing assistance transitioning to Stanage and Bessemer. You can access support through 
	- The `courses <https://sites.google.com/sheffield.ac.uk/research-training/>`_ we offer 

> 	- Our weekly drop in session 

Reviewer: Say how advertised?

	- Or by reaching out via email to ``researchcomputing@sheffield.ac.uk``  with your query. 

> - A huge thanks to the members of the team who contributed towards its creation in the early days and for the ones who saw it through to its decommissioning. We plan to ship it to Ukraine so it can have a second life supporting researchers.

Reviewer: Mention uni in Kyiv that it will be sent to?

GPU Benchmarks:
--------------------

> - At the beginning of the Academic year, we benchmarked the H100 on Stanage and shared our discoveries, available `here <https://notesrcg.blogspot.com/2023/08/Stanage-HPC-new-h100-gpus-available-benchmarking.html/>`_ . Our findings highlighted how different workloads benefit different GPUs. This led us to investigating the Nvidia L40s as a potential addition to our GPU offerings. Our colleague, Farhad Allian, detailed his findings in the following `blog post <https://www.blogger.com/blog/post/preview/343285122053095411/2534088968302114079/>`_. 

Reviewer: Maybe note that Farhad's work was done in Nov?

Recent HPC documentation updates:

> ---------------------------------

Reviewer: Link via Sphinx reference?

- `Graphical user interface sessions on Stanage <https://docs.hpc.shef.ac.uk/en/latest/hpc/flight-desktop.html#graphical-sessions-on-stanage/>`_
- The ShARC documentation section have been moved to the `Decommissioned Clusters <https://docs.hpc.shef.ac.uk/en/latest/decommissioned/index.html />`_ section.

> - `Displaying job status with seff <https://docs.hpc.shef.ac.uk/en/latest/referenceinfo/scheduler/SLURM/Common-commands/seff.html#seff/>`_  

Reviewer: More important to make users aware that email alerts on Stanage and Bessemer now include job efficiency info (with some limitations)?

New Packages and software
--------------------------

> - `Molpro <https://docs.hpc.shef.ac.uk/en/latest/stanage/software/apps/molpro.html/>`_ is now available on Stanage.

Reviewer: Any other recent additions to note?  the following modulefiles modified since 1st Nov:

```
2023-11-01-14-49 all/RASPA2/2.0.47-foss-2022b.lua
2023-11-01-16-59 all/RASPA2/2.0.41-foss-2020b.lua
2023-11-03-10-36 all/JAGS/4.3.1-foss-2022a.lua
2023-11-03-10-38 all/OpenJPEG/2.5.0-GCCcore-11.3.0.lua
2023-11-03-10-47 all/rjags/4-13-foss-2022a-R-4.2.1.lua
2023-11-03-17-16 all/LAPACK/3.9.1-GCC-10.2.0.lua
2023-11-07-12-29 all/MATLAB/2023b.lua
2023-11-10-16-48 all/PETSc/3.20.1-foss-2022b.lua
2023-11-14-13-36 all/VTK/9.2.2-foss-2022a.lua
2023-11-14-13-38 all/PICI-LIGGGHTS/3.8.1-foss-2022a.lua
2023-11-14-15-43 all/VCFtools/0.1.16-GCC-11.3.0.lua
2023-11-14-15-44 all/BCFtools/1.15.1-GCC-11.3.0.lua
2023-11-23-14-09 all/ADMIXTURE/1.3.0.lua
2023-11-23-15-26 all/SDL2/2.26.3-GCCcore-12.2.0.lua
2023-11-23-15-28 all/POV-Ray/3.7.0.7-foss-2022b.lua
2023-11-24-13-07 all/PETSc/3.17.4-foss-2022b.lua
2023-11-24-13-11 all/gmsh/4.11.1-foss-2022b.lua
2023-11-28-16-49 all/bzip2/1.0.6-GCCcore-7.3.0.lua
2023-11-28-16-49 all/libreadline/7.0-GCCcore-7.3.0.lua
2023-11-28-16-55 all/SQLite/3.24.0-GCCcore-7.3.0.lua
2023-11-28-16-55 all/Tcl/8.6.8-GCCcore-7.3.0.lua
2023-11-29-13-46 all/idl/8.8.lua
2023-11-30-09-52 all/libdeflate/1.8-GCCcore-11.2.0.lua
2023-11-30-09-53 all/HTSlib/1.14-GCC-11.2.0.lua
2023-11-30-09-53 all/StringTie/2.2.1-GCC-11.2.0.lua
2023-11-30-09-57 all/NGS/2.11.2-GCCcore-11.2.0.lua
2023-11-30-09-57 all/Perl/5.34.0-GCCcore-11.2.0-minimal.lua
2023-11-30-09-58 all/file/5.41-GCCcore-11.2.0.lua
2023-11-30-15-44 all/SCons/4.4.0-GCCcore-11.3.0.lua
2023-12-04-11-24 all/CASTEP/23.1-intel-2022a.lua
2023-12-12-08-40 all/rosetta/2023.35.lua
```

Reviewer: I think at least CASTEP, PETSc, MATLAB worth a mention here if documented and tested

Upcoming training:
------------------
Below are key training dates for January.

- Linux training - 11/01/2024
- Introduction to using HPC 18/01/2024

You can register for the above and more courses  `on this link <https://sites.google.com/sheffield.ac.uk/research-training/>`_ .

Planned Maintenance
--------------------

> - Stanage will be closed for maintenance from the 15th to the 18th of December.

Reviewer: ```suggestion
- Stanage will be entirely unavailable from 09:00 on 15th December to 17:00 on 19th of December due to essential maintenance.
```

> - Stanage will be closed for maintenance from the 15th to the 18th of December.

Reviewer 2: @nikishe Stanage will not be available until Tuesday 19th December 5pm

> - Abaqus license update on the 21st of December

Reviewer: Is this of interest to users?  Do they need to do anything about it?

> - Ansys license update end of December. 

Reviewer: Is this of interest to users?  Do they need to do anything about it?

Author: Des mentioned if it doesnt go seamlesly there tends to be some downtime. Though its very minimal

> - The team will be away for the festive period from the 23/12/23 to the 2/1/24

Reviewer: ```suggestion
- The team will be away over the festive period from 23/12/2023 to 2/1/2024 inclusive.
```

> - The team will be away for the festive period from the 23/12/23 to the 2/1/24

Reviewer 2: @nikishe I think I would put 3/1/2024 otherwise people might think we are working on the 2nd

Useful Links
-------------

> - `HPC documentation <https://docs.hpc.shef.ac.uk/en/latest/decommissioned/index.html/>`_

Reviewer: Link to home page via Sphinx link instead?

> - `RSE code clinic <https://rse.shef.ac.uk/support/code-clinic/>`_

Reviewer: Brief sentence on what CC's are for?

> - `Citing the HPC services in publications. <https://docs.hpc.shef.ac.uk/en/latest/citing.html#citing-the-hpc-services-in-publications/>`_

Reviewer: Link via Sphinx reference?

> - `women in HPC. <https://womeninhpc.org/>`_ 

Reviewer: ```suggestion
- `Women in HPC. <https://womeninhpc.org/>`_ organisation
```

.. _nl20231201:

> HPC Newsletter 2023-12-01

Reviewer: How about just
```suggestion
News - 2023-12-01
```
?

=========================

ShARC Decommissioned
---------------------
- As of November 30th, we have officially decommissioned our oldest High-Performance Computing (HPC) cluster, ShARC. For reference, we have archived ShARC's documentation :ref:`here <decommissioned>`.  
- Support remains available for users needing assistance transitioning to Stanage and Bessemer. You can access support through 
	- The `courses <https://sites.google.com/sheffield.ac.uk/research-training/>`_ we offer 
	- Join our drop-in sessions by signing up through the Google Form included in our weekly email that advertises these sessions.
	- Or by reaching out via email to ``researchcomputing@sheffield.ac.uk``  with your query. 

> - A huge thanks to the members of the team who contributed towards its creation in the early days and for the ones who saw it through to its decommissioning. We plan to ship it to `Igor Sikorsky Kyiv Polytechnic Institute` in kyiv, so that it can have a second life supporting researchers.

Reviewer: ```suggestion
- A huge thanks to the members of the team who contributed towards its creation in the early days and for the ones who saw it through to its decommissioning. We plan to ship it to the `Igor Sikorsky Kyiv Polytechnic Institute` in Kyiv, so that it can have a second life supporting researchers.
```

GPU Benchmarks:
--------------------

> - At the beginning of the Academic year, we benchmarked the H100 on Stanage and shared our discoveries, available `here <https://notesrcg.blogspot.com/2023/08/Stanage-HPC-new-h100-gpus-available-benchmarking.html>`_ . Our findings highlighted how different workloads benefit different GPUs. This led us to investigating the Nvidia L40s as a potential addition to our GPU offerings. Our colleagues, Farhad Allian and Peter Heywood, detailed their findings in the following `blog post <https://notesrcg.blogspot.com/2023/12/blog-post.html>`_ in November. 

Reviewer 3: ```suggestion
- At the beginning of the Academic year, we benchmarked the H100 on Stanage and shared our discoveries, available `here <https://notesrcg.blogspot.com/2023/08/Stanage-HPC-new-h100-gpus-available-benchmarking.html>`_ . Our findings highlighted how different workloads benefit from different GPUs. This led us to investigating the Nvidia L40s as a potential addition to our GPU offerings. Our colleagues, Farhad Allian and Peter Heywood, detailed their findings in the following `blog post <https://notesrcg.blogspot.com/2023/12/blog-post.html>`_ in November. 
```

> - At the beginning of the Academic year, we benchmarked the H100 on Stanage and shared our discoveries, available `here <https://notesrcg.blogspot.com/2023/08/Stanage-HPC-new-h100-gpus-available-benchmarking.html>`_ . Our findings highlighted how different workloads benefit different GPUs. This led us to investigating the Nvidia L40s as a potential addition to our GPU offerings. Our colleagues, Farhad Allian and Peter Heywood, detailed their findings in the following `blog post <https://notesrcg.blogspot.com/2023/12/blog-post.html>`_ in November. 

Reviewer: ```suggestion
- At the beginning of the academic year, we benchmarked the H100 on Stanage and shared our discoveries, available `here <https://notesrcg.blogspot.com/2023/08/Stanage-HPC-new-h100-gpus-available-benchmarking.html>`_ . Our findings highlighted how different workloads benefit different GPUs. This led us to investigating the Nvidia L40s as a potential addition to our GPU offerings. Our colleagues, Farhad Allian and Peter Heywood, detailed their findings in the following `blog post <https://notesrcg.blogspot.com/2023/12/blog-post.html>`_ in November. 
```

> - At the beginning of the Academic year, we benchmarked the H100 on Stanage and shared our discoveries, available `here <https://notesrcg.blogspot.com/2023/08/Stanage-HPC-new-h100-gpus-available-benchmarking.html>`_ . Our findings highlighted how different workloads benefit different GPUs. This led us to investigating the Nvidia L40s as a potential addition to our GPU offerings. Our colleagues, Farhad Allian and Peter Heywood, detailed their findings in the following `blog post <https://notesrcg.blogspot.com/2023/12/blog-post.html>`_ in November. 

Reviewer: ```suggestion
- At the beginning of the Academic year, we benchmarked the performance of NVIDIA H100 GPUs on Stanage and shared our discoveries  `here <https://notesrcg.blogspot.com/2023/08/Stanage-HPC-new-h100-gpus-available-benchmarking.html>`__. Our findings highlighted how different workloads benefit different GPUs. This led us to more recently investigating NVIDIA L40 GPUs as a potential addition to our GPU offerings. Our colleagues, Farhad Allian and Peter Heywood, detailed their findings in the following `blog post <https://notesrcg.blogspot.com/2023/12/blog-post.html>`_ in November. 
```

Recent HPC documentation updates:
---------------------------------
- :ref:`Graphical user interface sessions on Stanage <flight-desktop>`
- The ShARC documentation section has been moved to the :ref:`Decommissioned Clusters <decommissioned>` section.

> - Displaying job status with :ref:`seff  <seff>`. Job completion emails contain limited job efficiency information. Please make use of these to optimise your resource request.

Reviewer 3: ```suggestion
- Job completion emails contain limited job efficiency information, displaying job status with :ref:`seff  <seff>` gives more detail. Please make use of these to optimise your resource request.
```

New Packages and software
--------------------------

The following software are now available on Stanage:

> - :ref:`Castep 23.1  <castep_stanage>` 

Reviewer: ```suggestion
- :ref:`CASTEP 23.1  <castep_stanage>` 
```

> - :ref:`Matlab 2023b  <matlab_stanage>` 

Reviewer: ```suggestion
- :ref:`MATLAB 2023b  <matlab_stanage>` 
```

- :ref:`Molpro 2022.3.2  <molpro>` 
- :ref:`PETSc 3.20.1 && 3.17.4  <petsc_stanage>` 
- :ref:`Rosetta 2023.35  <rosetta_stanage>` 

Upcoming training:
------------------
Below are key training dates for January.

- Linux training - 11/01/2024
- Introduction to using HPC 18/01/2024

> You can register for the above and more courses  `on this link <https://sites.google.com/sheffield.ac.uk/research-training/>`_ .
> 

Reviewer 3: ```suggestion
You can register for the above and more courses by `following this link <https://sites.google.com/sheffield.ac.uk/research-training/>`_ .

```
